appcues/src/main/java/com/appcues/data/mapper/step/StepMapper.kt

```diff
@@ -19,16 +20,17 @@ internal class StepMapper(
 
     fun map(
         from: StepResponse,
-        superTraits: List<TraitResponse>,
-        superActions: Map<UUID, List<ActionResponse>>,
+        stepContainerTraits: List<LeveledTraitResponse>,
+        stepContainerActions: Map<UUID, List<ActionResponse>>,
     ): Step {
-        val mergedTraits = from.traits.mergeTraits(superTraits)
-        val mergedActions = from.actions.mergeActions(superActions)
+        val stepTraits = from.traits.map { it to STEP }
```

Comment:
and same here for Step level traits, then later we map and the mapper knows each trait level without extra parameter.